samples/apps/logging/logging.cpp

```diff
@@ -157,8 +169,10 @@ namespace loggingapp
             "Cannot record an empty log message.");
         }
 
-        auto records_handle = ctx.tx.rw(records);
+        // SNIPPET: private_table_access
+        auto records_handle = ctx.tx.template rw<RecordsTable>("records");
```

Comment:
Why changing those everywhere in this app? Having a unique reference to the map object seems like good practice for apps (avoid typo in table name, etc.).

Reply:
So this pattern of having a single instance of a `kv::Map` isn't something we're now doing in most places, so I don't think we should recommend it in the sample app. In particular, it was used in the docs to suggest this creates a `Map`, which is no longer true under the hood (does the distinction matter? Maybe not).

On the one hand, giving this repeated string a name to avoid typos seems like a good idea. On the other, I'm keen to reinforce to anyone reading this that a `Map` can be accessed from anywhere, purely by name, and hopefully with the same type. You don't need a single value (that could have some lookup magic under the hood), you can construct your string names on-the-fly. So I think there's value in having these redundant, and perhaps even not-best-practice, for clarity when reading a single endpoint? Arguable.

Reply:
Perhaps just defining the name of the table once instead of repeating the name of the table in multiple places is a good compromise?